WooCommerce/Classes/ViewModels/Refunds/RefundedProductsViewModel.swift

```diff
@@ -104,3 +104,22 @@ extension RefundedProductsViewModel {
         }
     }
 }
+
+
+// MARK: - Syncing data. Yosemite related stuff
+//
+extension RefundedProductsViewModel {
+    func syncOrder(onCompletion: ((Order?, Error?) -> ())? = nil) {
```

Comment:
nit: just wondering if it might make sense to move this responsibility of syncing an Order to the view controller? a "view model" feels like just taking a data model and providing data fields for the UI, but there could be other interpretations and ways of doing it